Dockerfile

```diff
@@ -11,12 +15,19 @@ RUN yum install -y \
 	db4-utils \
 	db4 && yum clean all
 
+RUN usermod -u ${USER_ID} ftp
+RUN groupmod -g ${GROUP_ID} ftp
+
 ENV FTP_USER **String**
 ENV FTP_PASS **Random**
 ENV PASV_ADDRESS **IPv4**
+ENV PASV_ADDR_RESOLVE No
+ENV PASV_ENABLE Yes
```

Comment:
Will need to document these on README, as users may want to change it when launching containers.

Reply:
Done